lib/jekyll-feed/feed.xml

```diff
@@ -51,6 +50,13 @@
     {% assign posts = posts | where_exp: "post", "post.draft != true" %}
   {% endunless %}
   {% assign posts = posts | sort: "date" | reverse %}
+  {% if site.feed.updated_deterministic %}
+    {% assign last_post = posts | first %}
+    {% assign last_updated = last_post.last_modified_at | default: last_post.date | default: site.time %}
```

Comment:
I don't mind supporting the `jekyll-last-modified-at` plugin, as it seems already supported by jekyll-feed. Good point on looping through, I'll amend the change to do that.

I think supporting it is relevant to the `<updated>` element. This value would otherwise be limited to original publishing dates only. I imagine people who care about this element to use my new option, would likely still want to increment it when they update existing posts, to ensure readers pick it up. Supporting this plugin, seems like a good option to offer for those people, so that at least there is a way to do that.

6000 dependents compared to 1 million for jekyll-feed is small indeed, but 6000 is a lot of sites. A loose [search for both](https://github.com/search?q=%22jekyll-last-modified-at%22+%22jekyll-feed%22&type=code) suggests there's at least 4000 sites with their source public on github.com using both. I imagine there's more usage among self-hosted sites, on other Git platforms, and in private repos.

Reply:
`last_modified_at` is already being used by jekyll-feed:

https://github.com/jekyll/jekyll-feed/blob/1ac64bcf074caefe4328f12e53cac6033e4bd29e/lib/jekyll-feed/feed.xml#L62

It was added in 2015 with a dependency to jekyll-last-modified-at, but this dependency was removed in 2016. Now it relies on manually setting `last_modified_at` in frontmatter:

https://github.com/jekyll/jekyll-feed/search?q=last_modified_at&type=commits

`last_modified_at` is also supported by jekyll-seo-tag:

https://github.com/jekyll/jekyll-seo-tag/search?q=last_modified_at&type=code

IMHO jekyll-feed shouldn't stop using it and may use it for additional features.